docs/proposals/workload.md

```diff
@@ -0,0 +1,81 @@
+# Workload schema
+
+Example :
+
+```json
+{
+    "kind" : "workload",
+    "type": "container",
+    "name": "test-workload",
+    "uri": "nginx",
+    "ressources": {
+        "cpu": 1.0,
+        "ram": 2048,
+        "disk": 2,
+    },
+    "ports": [
+        "8080:80"
+    ],
+    "environment": [
+        "NODE_ENV=production"
+    ]
```

Comment:
Just curious, why define a string list instead of a map where the key is the environment variable name and the value... its value?

Reply:
Puting everything in one string allows to use the Unix syntax `<key>=<value>` for environment variables whereas a map uses a less user-friendly notation `"<key>":"<value>" in JSON and `<key>: <value>` in YAML with the need to put a space between the : and the value.